changelog/0082-sms-gateway-changes.yml

```
@@ -0,0 +1,35 @@
+title: sms-gateway-changes
+key: sms-gateway-changes
```

I realize this will require renaming files, but I think a more descriptive name is a good idea.

```suggestion
title: Unicel SMS Gateway Removal
key: unicel-sms-gateway-removal
```